sympy/printing/printer.py

```diff
@@ -285,7 +285,11 @@ def _print(self, expr, **kwargs):
                 printmethod = '_print_' + cls.__name__
                 if hasattr(self, printmethod):
                     return getattr(self, printmethod)(expr, **kwargs)
-            # Unknown object, fall back to the emptyPrinter.
+            # Unknown object, fall back to the emptyPrinter. Checks what type of
+            # decimal separator to print.
+            if (self.emptyPrinter == str) & \
+                (self._settings.get('decimal_separator', None) == 'comma'):
+                expr = str(expr).replace('.', '{,}')
```

Comment:
Why is this code in the printer superclass? This should go in the individual printers for Float.

Reply:
I already have a PR for this